test/fixtures/votes.yml

```
@@ -0,0 +1,15 @@
+vote_movie:
+  user_id: test_user
+  work_id: movie
```

You don't specify the id just note this this refers to a user or work.

```suggestion
  user: test_user
  work: movie
```